### PR TITLE
HPCC-19088 Document Std TimeZone functions

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@dcamper Please review (pdf sent via email)

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/FindTZData.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+          <entry>OPTIONAL. The name of the location to search for; if a
+          location is not provided or is an empty string, all records matching
+          only the abbreviation are returned.</entry>
+        </row>
+
+        <row>
+          <entry>Returns:</entry>
+
+          <entry>A new DATASET(TZDataLayout) containing the found
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/FindTZData.xml]: Recommend adding the record definition for TZDataLayout somewhere on this page (similar to what was done for Std.File.SuperfileContents).

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/AdjustTimeTZ.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>time</emphasis></entry>
+
+          <entry>REQUIRED. The time value to adjust.</entry>
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/AdjustTimeTZ.xml]: Note that the data type is Std.Date.Time_t.

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToLocalTime.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>utcTime</emphasis></entry>
+
+          <entry>REQUIRED. The UTC time value to adjust.</entry>
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToLocalTime.xml]: Note that the data type is Std.Date.Time_t.

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToUTCTime.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>localTime</emphasis></entry>
+
+          <entry>REQUIRED. The time value to adjust.</entry>
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToUTCTime.xml]: Note that the data type is Std.Date.Time_t.

**Review by dcamper — CHANGES_REQUESTED** [review]
Looks very good.  A few small changes.

**Comment by dcamper** [comment]
@JamesDeFabia Looks great, overall.  Just a few minor changes.

**Comment by JamesDeFabia** [comment]
@dcamper requested changes made, back to you

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/AdjustTimeTZ.xml [hunk 5]
```diff
@@ -26,15 +26,16 @@
...
+          adjust.</entry>
         </row>
 
         <row>
           <entry><emphasis>fromTimeZoneAbbrev</emphasis></entry>
 
           <entry>REQUIRED. The time zone abbreviation that the
           <emphasis>time</emphasis> value is assumed to be within; must be a
-          non-empty uppercase string; </entry>
+          non-empty uppercase string;</entry>
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/AdjustTimeTZ.xml]: Trailing semicolon.

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToUTCTime.xml [hunk 6]
```diff
@@ -26,15 +26,16 @@
...
         </row>
 
         <row>
           <entry><emphasis>fromTimeZoneAbbrev</emphasis></entry>
 
           <entry>REQUIRED. The time zone abbreviation that the localTime value
-          is assumed to be within; must be a non-empty uppercase string;
-          </entry>
+          is assumed to be within; must be a non-empty uppercase
+          string;</entry>
```
Comment (dcamper) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/ToUTCTime.xml]: Trailing semicolon.

**Review by dcamper — CHANGES_REQUESTED** [review]
@JamesDeFabia Two minor things I missed first time around, then I think we'll be done.

**Comment by JamesDeFabia** [comment]
@dcamper back to you. Once you approve, i will squash

**Review by dcamper — APPROVED** [review]
@JamesDeFabia Looks great; can merge.

**Comment by JamesDeFabia** [comment]
@richardkchapman This is good to merge. If you want me to squash, let me know

**Comment by richardkchapman** [comment]
@JamesDeFabia Please squash

**Pull request merged**